components/script/dom/performancenavigationtiming.rs

```diff
@@ -0,0 +1,130 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use dom::bindings::codegen::Bindings::PerformanceBinding::DOMHighResTimeStamp;
+use dom::bindings::codegen::Bindings::PerformanceNavigationTimingBinding;
+use dom::bindings::codegen::Bindings::PerformanceNavigationTimingBinding::{PerformanceNavigationTimingMethods, NavigationType};
+use dom::bindings::codegen::Bindings::WindowBinding::WindowMethods;
+use dom::bindings::num::Finite;
+use dom::bindings::reflector::{Reflector, reflect_dom_object};
+use dom::bindings::root::{Dom, DomRoot};
+use dom::bindings::str::DOMString;
+use dom::document::Document;
+use dom::performanceresourcetiming::PerformanceResourceTiming;
+use dom::window::Window;
+use dom_struct::dom_struct;
+
+#[dom_struct]
+ // https://w3c.github.io/navigation-timing/#dom-performancenavigationtiming
+pub struct PerformanceNavigationTiming {
+    reflector_: Reflector,
+    // https://w3c.github.io/navigation-timing/#PerformanceResourceTiming
+    resource_timing: Dom<PerformanceResourceTiming>,
```

Comment:
This needs to be `performanceresourcetiming: PerformanceResourceTiming` instead of `relector_` and `resource_timing` ([docs](https://doc.servo.org/script/dom/index.html#inheritance)).